tests/Model/StubsContainer.php

```diff
@@ -126,6 +126,11 @@ public function getFunction($name, $sourceFilePath = null, $shouldSuitCurrentPhp
         if (!empty($functions)) {
             return array_pop($functions);
         }
+        // Added this while tracking down "Attempt to read property "parameters" on null" error.
+        // Happens when BasePHPElement::entitySuitsCurrentPhpVersion returns false when running tests on PHP beta: getenv('PHP_VERSION')
+        if (!$functions) {
+            throw new RuntimeException("Could not get function " . $name . " from reflection, possibly unsupported PHP_VERSION in getenv");
```

Comment:
I think concatenation can be replaced with interpolation here since it's just one simple variable